webroot/src/network/userApi/interceptors.ts

```diff
@@ -17,7 +17,7 @@ export const requestSuccess = () => async (requestConfig) => {
     // Endpoints in the below list are accessible to users who are logged in via the licensee userpool
     // and as such require sending that token to authorize their use. All other endpoints are only accessible
     // by users logged in via the staff user pool and will therefor send those tokens to authorize.
```

Comment:
just noticed small typo: 
```suggestion
    // by users logged in via the staff user pool and will therefore send those tokens to authorize.
```